<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="VISAI00215L" failures="3" tests="9" name="API Tests - ListMeasurementIDandDates" time="42.518" errors="0" timestamp="2023-09-08T19:37:12 IST">
  <testcase classname="v1.Get.api.ListMeasurementIDandDates" name="Notfound" time="1.425"/>
  <testcase classname="v1.Get.api.ListMeasurementIDandDates" name="ScanIdListEndpoint" time="14.787">
    <failure type="java.lang.AssertionError" message="&amp;apos;id&amp;apos; should be an integer expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: 'id' should be an integer expected [true] but found [false]
at credentails.CommonMethods.assertIdAndDate(CommonMethods.java:178)
at v1.Get.api.ListMeasurementIDandDates.ScanIdListEndpoint(ListMeasurementIDandDates.java:26)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- ScanIdListEndpoint -->
  <testcase classname="v1.Get.api.ListMeasurementIDandDates" name="Unauthorized" time="1.381"/>
  <testcase classname="v2.Get.api.ListMeasurementIDandDates" name="Notfound" time="1.3"/>
  <testcase classname="v2.Get.api.ListMeasurementIDandDates" name="ScanIdList" time="9.949">
    <failure type="java.lang.AssertionError" message="&amp;apos;id&amp;apos; should be an integer expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: 'id' should be an integer expected [true] but found [false]
at credentails.CommonMethods.assertIdAndDate(CommonMethods.java:178)
at v2.Get.api.ListMeasurementIDandDates.ScanIdList(ListMeasurementIDandDates.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- ScanIdList -->
  <testcase classname="v2.Get.api.ListMeasurementIDandDates" name="Unauthorized" time="1.34"/>
  <testcase classname="v3.Get.api.ListMeasurementIDandDates" name="Notfound" time="1.443"/>
  <testcase classname="v3.Get.api.ListMeasurementIDandDates" name="Unauthorized" time="1.278"/>
  <testcase classname="v3.Get.api.ListMeasurementIDandDates" name="testScanIdListEndpoint" time="9.602">
    <failure type="java.lang.AssertionError" message="&amp;apos;id&amp;apos; should be an integer expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: 'id' should be an integer expected [true] but found [false]
at credentails.CommonMethods.assertIdAndDate(CommonMethods.java:178)
at v3.Get.api.ListMeasurementIDandDates.testScanIdListEndpoint(ListMeasurementIDandDates.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- testScanIdListEndpoint -->
</testsuite> <!-- API Tests - ListMeasurementIDandDates -->
